Comment out some tests that require open_memstream (function added in 10.13)

--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -116,16 +116,16 @@ target_link_libraries(rastertobrlaser ${CUPS_LDFLAGS})
 add_executable(brdecode EXCLUDE_FROM_ALL src/brdecode.cc)
 add_executable(test_lest test/test_lest.cc)
 add_executable(test_line test/test_line.cc src/line.cc)
-add_executable(test_block test/test_block.cc)
+# add_executable(test_block test/test_block.cc)
 
 enable_testing()
 add_test(test_lest test_lest)
 add_test(test_line test_line)
-add_test(test_block test_block)
+# add_test(test_block test_block)
 
 # Autotools-style "make check" command
 add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
-add_dependencies(check test_lest test_line test_block)
+add_dependencies(check test_lest test_line) # test_block
 
 
 # Installation
